### PR TITLE
Article prototype example

### PR DESCRIPTION
**Pull request by tylersticka**
## Overview

Introduces a new prototypes section including a single Article prototype example. This example is meant as a starting point for others to "save as" and change, it isn't a proposed design.

The feed data was harvested from our RSS feed but with changes applied for prototyping based on JSON feed format.

## Screenshots

Screenshot_2020-04-09 Prototypes Articles - Example ⋅ Storybook

---

- See #477 #493

### CODE REVIEW

File: src/prototypes/articles/example/example.twig [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    {% for article in feed.items %}
+      <article>
+        <header>
+          <h2>
+            <a href="{{article.url}}">
+              {{article.title}}
+            </a>
+          </h2>
+        </header>
+        <p>
```
Comment (derekshirk) [inline src/prototypes/articles/example/example.twig:12]: 🤔 Would it be appropriate to wrap content after `<header>` and before `<footer>` in a `<section>` tag?

Comment (tylersticka) [reply src/prototypes/articles/example/example.twig:12]: No, `<section>` would not be appropriate there.

Comment (derekshirk) [reply src/prototypes/articles/example/example.twig:12]: Gotcha, I saw [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#Examples) example and wasn't sure.

Comment (tylersticka) [reply src/prototypes/articles/example/example.twig:12]: That example confuses me.

The good news is that this is just an example: For prototyping purposes, you can "save as" and write whatever markup you want! 🎉

Comment (derekshirk) [reply src/prototypes/articles/example/example.twig:12]: 😄 Thank you sir.

**Pull request merged**